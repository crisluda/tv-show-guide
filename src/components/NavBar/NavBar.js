import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav>
    <Link
      to="/"
      style={{
        marginLeft: '50px',
        fontSize: '40px',
        fontWeight: 'bold',
        fontFamily: 'Dancing Script, cursive',
      }}
    >
      Cars Show Guide
    </Link>
  </nav>
);

export default Navbar;
