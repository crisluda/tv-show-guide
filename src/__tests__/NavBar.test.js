import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../components/NavBar/NavBar';

describe('render NavBar with link correctly', () => {
  it('should match with snapshot', () => {
    const tree = render(<BrowserRouter><NavBar /></BrowserRouter>); expect(tree).toMatchSnapshot();
  });
});
