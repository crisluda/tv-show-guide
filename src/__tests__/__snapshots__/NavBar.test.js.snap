// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NavBar with link correctly should match with snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <a
          href="/"
          style="margin-left: 50px; font-size: 40px; font-weight: bold; font-family: Dancing Script, cursive;"
        >
          Cars Show Guide
        </a>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav>
      <a
        href="/"
        style="margin-left: 50px; font-size: 40px; font-weight: bold; font-family: Dancing Script, cursive;"
      >
        Cars Show Guide
      </a>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
